### PR TITLE
Raising line length limit

### PR DESCRIPTION
**Pull request by imjacobclark**

### CODE REVIEW

**Comment by cathawker** [comment]
LGTM

**Comment by stuartmemo** [comment]
:no_good: Needs more discussion.

**Comment by katie-roberts** [comment]
@stuartmemo - an email was sent to the unified place developers list at at 11.15am today about this - if you have discussion points could you add it to this thread?

**Comment by stuartmemo** [comment]
@katie-roberts We've been discussing it on Slack.

**Comment by imjacobclark** [comment]
As discussed, 100.

Are we okay with this?

File: .jshintrc [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "eqeqeq"        : true,     // Prohibit use of == and != in favor of === and !==.
     "indent"        : 4,        // Enforce tab width of 4 spaces.
     "latedef"       : true,     // Prohibit use of a variable before it is defined.
-    "maxlen"        : 80,       // Enforce line length to 80 characters
+    "maxlen"        : 100,       // Enforce line length to 80 characters
```
Comment (randombitsUK) [inline .jshintrc]: The comment needs updated too...

**Comment by djorak** [comment]
Looks good to me.

**Comment by intih** [comment]
Yep - feel free to merge.

**Pull request merged**